src/PaymentGateways/PayPalCommerce/PayPalCommerce.php

```diff
@@ -14,36 +19,106 @@
  *
  * @since 2.9.0
  */
-class PayPalCommerce implements PaymentGateway
+class PayPalCommerce extends PaymentGateway
 {
+    /**
+     * @deprecated
+     *
+     * Use getId or id function to access payment gateway id.
+     *
+     */
     const GATEWAY_ID = 'paypal-commerce';
 
     /**
-     * @inheritDoc
+     * @unreleased
+     *
+     * @param int $formId
+     * @param array $args
+     *
+     * @return string
+     */
+    public function getLegacyFormFieldMarkup($formId, $args)
+    {
+        ob_start();
```

Comment:
I might suggest moving the output buffer to the method that generates the markup. I could see it either way, though.

Reply:
fixed: https://github.com/impress-org/givewp/compare/f5056d6cecd4...d3c2f8238acf